### ISSUE TITLE
Sub-suites are still executed after a failure

### ISSUE THREAD

**Issue by tandrup** [rprieto/mocha-steps#7]
If I have a sub suite after a `step` it still tries to run the tests in the sub suite.

```javascript
describe('Dummy', function() {
    step('should not be ok', function() {
        expect(true).to.be.false;
    });
    
    describe('sub stuff', function() {
        step('should not run', function() {
        });
    });
});
```

**Comment by mogsie** [comment]
I can verify the usefulness of PR 8, it works as intended. It makes it easy to make tests that pinpoint the problem when they fail. Very useful.

**Comment by rprieto** [comment]
Thanks @mogsie. Just merged PR 8 which is available as `v0.5.0`
